app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__TargetGroupShow.re

```diff
@@ -132,15 +132,17 @@ let make =
           <h4> {targetGroup |> TargetGroup.name |> str} </h4>
         </div>
         <div className="target-group__description">
-          <p className="pt-px text-sm">
-            {(
-               switch (targetGroup |> TargetGroup.description) {
-               | Some(description) => description
-               | None => ""
-               }
-             )
-             |> str}
-          </p>
+          {let description =
+             switch (targetGroup |> TargetGroup.description) {
+             | Some(description) => description
+             | None => ""
+             };
```

Comment:
1. Is the container `div class="target-group__description` above this really required even if there is no description? Wouldn't it be cleaner to avoid placing the `div` in the markup in this case?
2. If it is required, it would be cleaner to use `Belt.Option.getWithDefault` to set the `description` binding.
   ```reason
   let description = TargetGroup.description(targetGroup)->Belt.Option.getWithDefault("")
   ```